drizzle-kit/build.dev.ts

```diff
@@ -24,7 +24,7 @@ esbuild.buildSync({
 	platform: 'node',
 	external: ['drizzle-orm', 'esbuild', ...driversPackages],
 	banner: {
-		js: `#!/usr/bin/env -S node --loader @esbuild-kit/esm-loader --no-warnings`,
+		js: '#!/usr/bin/env -S npx tsx',
```

Comment:
npm (therefore npx) is bundled with all node installations so I expect this to always work.

If anyone disagrees then I can switch this. line to `node --include tsx`

Reply:
my guess is that `node --include tsx` should be (a tiny bit) faster: we invoke `node` binary in a more straightforward way.

On the other side I would be glad to see this merged and released in any form.

Reply:
I wasn't able to make it work with `node --include tsx`, but it works fine with `node --import tsx`, maybe that's the one you referred to? I tested it across all Node versions between 18 and 24.

I agree with that it should call node directly 😄 Let’s keep the original param as well, since tsx is able to handle `--no-warnings` as per [their docs](https://tsx.is/node-enhancement#swap-node-for-tsx):

```suggestion
		js: '#!/usr/bin/env -S node --import tsx --no-warnings',
```